src/main/java/org/aksw/word2vecrestful/subset/DataSubsetProvider.java

```diff
@@ -0,0 +1,70 @@
+package org.aksw.word2vecrestful.subset;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.aksw.word2vecrestful.utils.Cfg;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+/**
+ * Class to help retrieve the list of words from a subset stored on the disk
+ * 
+ * @author Nikit
+ *
+ */
+public class DataSubsetProvider {

```

Comment:
I am not sure why you are creataing JSON files when you want to write a list of words. Why not simply writing one word per line? There are FileUtils methods that can do that very easily ;)